extensions/3rdparty/tabber/tabber.php

```diff
@@ -57,18 +57,18 @@ function renderTabber( $paramstring, $params, $parser ){
  * @param Parser $parser
  * @return string
  */
-function buildTab($tab, $parser){
-	if( trim($tab) == '' ) return '';
+function buildTab( $tab, $parser ) {
+	if ( trim( $tab ) == '' ) return '';
 
-	$arr = explode("=",$tab);
+	$arr = explode( "=", $tab );
 	$tabName = array_shift( $arr );
 	// Wikia Change Start @author aquilax
 	$tabName = trim( $tabName );
 	// Wikia Change End
-	$tabBody = $parser->recursiveTagParse( implode("=",$arr) );
+	$tabBody = $parser->recursiveTagParse( implode( "=", $arr ) );
 
-	$tab = '<div class="tabbertab" title="'.htmlspecialchars($tabName).'">'
-		. '<p>'.$tabBody.'</p>'
+	$tab = '<div class="tabbertab" title="' . htmlspecialchars( $tabName ) . '">'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$tab". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)